### DISCUSSION TITLE
Need help! TypeError: Cannot read properties of undefined (reading 'keyboard')

### DISCUSSION THREAD

**Discussion by romcik220886** [axios/axios#5554] [General]
Hello guys. Need help with Node.js code. I don't understand what is wrong here. 

Error message on bot start:

Unhandled error while processing {
  update_id: 146451130,
  message: {
    message_id: 2219,
    from: {
      id: 5884595232,
      is_bot: false,
      first_name: '®|v|',
      username: 'zeR0manners',
      language_code: 'ru'
    },
    chat: {
      id: 5884595232,
      first_name: '®|v|',
      username: 'zeR0manners',
      type: 'private'
    },
    date: 1676488445,
    text: '/start',
    entities: [ [Object] ]
  }
}
C:\Users\r2g\Desktop\Bot1\index.js:12
    const keyboard = Telegraf.Markup.keyboard([
                                     ^

TypeError: Cannot read properties of undefined (reading 'keyboard')
    at C:\Users\r2g\Desktop\Bot1\index.js:12:38
    at C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:167:111
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async execute (C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:474:17)
    at async C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:475:21
    at async execute (C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:474:17)
    at async C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:475:21
    at async execute (C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:474:17)
    at async C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:475:21
    at async execute (C:\Users\r2g\Desktop\Bot1\node_modules\telegraf\lib\composer.js:474:17)

Node.js v18.14.0

Part of code: 

bot.command('start', (ctx) => {
    **const keyboard = Telegraf.Markup.keyboard**([
        ['Suche'],
        ['Dashboard'],
        ['Crown Autos', 'Big Step Autos'],
        ["Gruppe Crown Rent", "Gruppe Big Step"]
    ]).resize().oneTime().extra();
    ctx.reply('Hallo, ich bin ein CR&BS-Bot. Wählen Sie ein Element aus dem Menü, was Sie tun möchten.', keyboard);
});
bot.hears('Suche', (ctx) => {
    const keyboard3 = Telegraf.Markup.keyboard([
        ['Suche'],
    ]).resize().extra();
    ctx.reply('Bitte Fahrzeug Name oder Fahrgestellnummer eingeben', keyboard3);
});

Please help guys.

**Comment by jasonsaayman** [comment]
This is unreadable please post with better info and formatting.
